web/bitwarden/bitwarden.go

```diff
@@ -51,6 +51,39 @@ func GetProfile(c echo.Context) error {
 	return c.JSON(http.StatusOK, profile)
 }
 
+// UpdateProfile is the handler for the route to update the profile. Currently,
+// only the hint for the master password can be changed.
+func UpdateProfile(c echo.Context) error {
+	inst := middlewares.GetInstance(c)
+	if err := middlewares.AllowWholeType(c, permission.POST, consts.BitwardenProfiles); err != nil {
```

Comment:
I'm not really familiar with permissions but shouldn't it be `permissions.PUT` here?